scripts/functions/pkg

```diff
@@ -279,8 +279,8 @@ install_openssl()
 update_openssl_certs()
 (
   __rvm_cd "$rvm_usr_path/ssl" &&
-  command curl -O http://curl.haxx.se/ca/cacert.pem &&
-  \command \mv cacert.pem cert.pem
+  command curl -O https://raw.githubusercontent.com/bagder/ca-bundle/master/ca-bundle.crt &&
```

Comment:
Oh right, It was unclear that there was a link between that github user and
the curl upstream (Sorry on planeternets).

My gut feel is that vendoring it is probably least wrong.

On Sun, Jul 19, 2015 at 11:22 AM, Reed Loden <notifications@github.com>
wrote:

> In scripts/functions/pkg
> <https://github.com/rvm/rvm/pull/3469#discussion_r34960561>:
>
> > @@ -279,8 +279,8 @@ install_openssl()
> >  update_openssl_certs()
> >  (
> >    __rvm_cd "$rvm_usr_path/ssl" &&
> > -  command curl -O http://curl.haxx.se/ca/cacert.pem &&
> > -  \command \mv cacert.pem cert.pem
> > +  command curl -O https://raw.githubusercontent.com/bagder/ca-bundle/master/ca-bundle.crt &&
>
> Can you explain? The current version uses http:// to download a list of
> CA roots with no type of authenticity checking at all, making it easy for
> somebody to MITM that URL to add a malicious CA root. I just replace it
> with an official https:// version (as per
> http://curl.haxx.se/docs/caextract.html). Of course, there is a slight
> chicken and egg problem here, as by using https:// to download the CA
> roots, you are requiring that curl itself has a proper CA root repository
> enough to download something from GitHub. However, considering the link to
> download & install RVM is over https:// as well, I think that's
> acceptable. Sadly, there's no GPG-signed version of this file that we could
> use for validation (similar to what https://rvm.io/rvm/security does,
> though that page recommends you download a GPG public key insecurely,
> sigh). Considering how frequently RVM is upd ated and released, another
> option is to actually include the CA bundle itself as part of RVM. However,
> not sure that's wanted... Any further comments/thoughts are appreciated.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rvm/rvm/pull/3469/files#r34960561>.
>